### ISSUE TITLE
[bug] `converter.py` does not convert rule correctly which has `contains` and `the last two character backslash`

### ISSUE THREAD

**Issue by fukusuket** [Yamato-Security/hayabusa-rules#392]
**Describe the bug**
`converter.py` does not convert rule correctly which has `contains` and `the last two character backslash` 🤔

```
e.g.
detection:
    selection:
        CommandLine|contains: \\\\.\\pipe\\
```

**Step to Reproduce**

1. git clone https://github.com/Yamato-Security/hayabusa-sample-evtx
2. `/hayabusa csv-timeline -d ../hayabusa-sample-evtx -r ./rules/sigma/builtin/process_creation/proc_creation_win_susp_priv_escalation_via_named_pipe.yml`
    - [proc_creation_win_susp_priv_escalation_via_named_pipe.yml](https://github.com/Yamato-Security/hayabusa-rules/blob/2eb4bde3afd549b4aa3191dccee323cf844dc52b/sigma/builtin/process_creation/proc_creation_win_susp_priv_escalation_via_named_pipe.yml#L4) use `contains` and `the last character backslash`

**Expected behavior**
The detection count is `5` and output following logs.
```
Timestamp ‖ Computer ‖ Channel ‖ EventID ‖ Level ‖ RecordID ‖ RuleTitle ‖ RuleFile ‖ Details
2016-08-19 05:03:18.175 +09:00 ‖ IE10Win7 ‖ Sec ‖ 4688 ‖ high ‖ 6749 ‖ Privilege Escalation via Named Pipe Impersonation ‖ proc_creation_win_susp_priv_escalation_via_named_pipe.yml ‖ Cmd: cmd.exe /c echo yagfag > \\.\pipe\yagfag ¦ Proc: C:\Windows\System32\cmd.exe ¦ PID: 0x57c ¦ User: IE10WIN7$ ¦ LID: 0x3e7
2016-09-20 06:12:10.682 +09:00 ‖ IE10Win7 ‖ Sec ‖ 4688 ‖ high ‖ 12838 ‖ Privilege Escalation via Named Pipe Impersonation ‖ proc_creation_win_susp_priv_escalation_via_named_pipe.yml ‖ Cmd: cmd.exe /c echo adsymn > \\.\pipe\adsymn ¦ Proc: C:\Windows\System32\cmd.exe ¦ PID: 0x3dc ¦ User: IE10WIN7$ ¦ LID: 0x3e7
2016-09-20 06:13:04.094 +09:00 ‖ IE10Win7 ‖ Sec ‖ 4688 ‖ high ‖ 12843 ‖ Privilege Escalation via Named Pipe Impersonation ‖ proc_creation_win_susp_priv_escalation_via_named_pipe.yml ‖ Cmd: cmd.exe /c echo hmoopk > \\.\pipe\hmoopk ¦ Proc: C:\Windows\System32\cmd.exe ¦ PID: 0xc10 ¦ User: IE10WIN7$ ¦ LID: 0x3e7
2016-09-21 01:35:58.169 +09:00 ‖ IE10Win7 ‖ Sec ‖ 4688 ‖ high ‖ 13283 ‖ Privilege Escalation via Named Pipe Impersonation ‖ proc_creation_win_susp_priv_escalation_via_named_pipe.yml ‖ Cmd: cmd.exe /c echo genusn > \\.\pipe\genusn ¦ Proc: C:\Windows\System32\cmd.exe ¦ PID: 0xe8c ¦ User: IE10WIN7$ ¦ LID: 0x3e7
2016-09-21 12:41:13.078 +09:00 ‖ IE10Win7 ‖ Sec ‖ 4688 ‖ high ‖ 13680 ‖ Privilege Escalation via Named Pipe Impersonation ‖ proc_creation_win_susp_priv_escalation_via_named_pipe.yml ‖ Cmd: cmd.exe /c echo hgabms > \\.\pipe\hgabms ¦ Proc: C:\Windows\System32\cmd.exe ¦ PID: 0x694 ¦ User: IE10WIN7$ ¦ LID: 0x3e7
```

**Actual behavior**
The detection count is `0`.

**Environment**

- OS:  `macOS montery version 13.1`
- hayabusa version ~`v.2.4.0`
  - I think `converter.py` has not changed so much, it seems to be an issue since the initial release.

**Additional context**
As far as I have checked, the rules affected by this issue in the [Sigma repo](https://github.com/SigmaHQ/sigma) are:
- [MSI Installation From Suspicious Locations](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules/windows/builtin/application/msiinstaller/win_msi_install_from_susp_locations.yml#L30)
- [Privilege Escalation via Named Pipe Impersonation](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules/windows/process_creation/proc_creation_win_susp_priv_escalation_via_named_pipe.yml#L31)
- [Wscript Execution from Non C Drive](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules/windows/process_creation/proc_creation_win_susp_lolbin_non_c_drive.yml#L36)
- [HackTool - Potential Impacket Lateral Movement Activity](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules/windows/process_creation/proc_creation_win_hktl_impacket_lateral_movement.yml#L53)
- [Suspicious Cmd Execution via WMI](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/deprecated/windows/proc_creation_win_susp_cmd_exectution_via_wmi.yml#L20)
- [Loading Diagcab Package From Remote Path](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules/windows/builtin/diagnosis/scripted/win_diagnosis_scripted_load_remote_diagcab.yml#L18)
- [APT31 Judgement Panda Activity](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules-emerging-threats/2019/TA/APT31/proc_creation_win_apt_apt31_judgement_panda.yml#L27)
- [Suspicious IO.FileStream](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules/windows/powershell/powershell_script/posh_ps_susp_iofilestream.yml#L22)
- [Suspicious New-PSDrive to Admin Share](https://github.com/SigmaHQ/sigma/blob/796fa721fd9961c86e91f2386fdfa6ffa9111e6e/rules/windows/powershell/powershell_script/posh_ps_susp_new_psdrive.yml#L24)
- ... etc

**Cause**
`converter.py` converts following rule 
```
detection:
    selection:
        CommandLine|contains: \\\\.\\pipe\\
```
into (`|contains` are converted into wildcard pattern internally)
```
detection:
    selection:
        CommandLine: '*\\\\.\\pipe\\\*
```

`\\\*` matches a single backslash and a literal asterisk(not interpreted as wildcard pattern)

ref:
[Sigma_specification.md#escaping](https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#escaping) 
>Three backslashes are necessary to escape both, the backslash and the wildcard and handle them as plain values: `\\\*`

**Comment by fukusuket** [comment]
This issue will be fixed in below branch(for #340)
- https://github.com/Yamato-Security/hayabusa-rules/compare/main...340-feat-keep-x-of-selection-as-is

**Comment by hitenkoku** [comment]
Thanks for contacting me with the relevant branch information.


I have made the association on Develop's end.
